### PR TITLE
New Atomic - Privilege Escalation via Docker Volume Mapping

### PR DESCRIPTION
**Pull request by krdmnbrk**
**Details:**
This atomic test demonstrates a privilege escalation attack leveraging Docker's volume mapping feature. By mounting the root directory of the host into a Docker container, the attacker can use the `chroot `command to access the host system as root. The test highlights the risks associated with granting excessive permissions to Docker users. It includes commands for setting up and cleaning up the environment, ensuring accurate execution and proper cleanup.

**Testing:**

- Verified that the Docker installation is detected as a prerequisite.
- Confirmed that the user running the test has the required NOPASSWD permissions for Docker.
- Successfully executed the command to mount the host root directory and escalate privileges using chroot.
- Ensured the cleanup command removes any test-created users and configurations.

Screenshot:

image

### CODE REVIEW

**Comment by krdmnbrk** [comment]
Hey @patel-bhavin , I am not sure about the supported platforms. I choose "container" but it is actually about privesc of linux machine, it just use docker to privesc. Maybe I can put both of linux and container platform. What do you think?

File: atomics/T1611/T1611.yaml [hunk 1]
```diff
@@ -143,3 +143,39 @@ atomic_tests:
...
+    - description: Docker Privileged User
+      prereq_command: |
+        sudo -l -U #{username} | grep "(ALL) NOPASSWD: /usr/bin/docker"
+      get_prereq_command: |
+        USERNAME="#{username}"
+        PASSWORD="password123"
+        SUDO_COMMAND="/usr/bin/docker"
+        SUDOERS_FILE="/etc/sudoers.d/$USERNAME"
+        [[ $EUID -ne 0 ]] && echo "Run as root." && exit 1; id "$USERNAME" &>/dev/null || { useradd -m -s /bin/bash "$USERNAME" && echo "$USERNAME:$PASSWORD" | chpasswd; }; [[ -f "$SUDOERS_FILE" ]] || { echo "$USERNAME ALL=(ALL) NOPASSWD: $SUDO_COMMAND" > "$SUDOERS_FILE" && chmod 440 "$SUDOERS_FILE"; }; echo "Setup complete. User: $USERNAME, Password: $PASSWORD"
+  input_arguments:
```
Comment (patel-bhavin) [inline atomics/T1611/T1611.yaml:177]: Can we perhaps  rearrange  the keys in the following order :


supported_platforms 
input_arguments
dependency_executor_name
dependencies
executor

to keep the atomics in a consistent key order. Also, I think for now lets leave it to `containers`

Comment (krdmnbrk) [reply atomics/T1611/T1611.yaml]: Thanks @patel-bhavin, done.

**Comment by patel-bhavin** [comment]
Neat! thank you for the atomic and the prompt fixes! Also, @krdmnbrk : Get it if you haven't already, we appreciate your contribution! 

https://github.com/redcanaryco/atomic-red-team/wiki/Contributing#claim-your-free-t-shirt

**Review by patel-bhavin — APPROVED** [review]

**Pull request merged**